src/components/v2v3/shared/SplitItem/SplitAmountValue.tsx

```diff
@@ -60,7 +60,9 @@ export function SplitAmountValue({ props }: { props: SplitProps }) {
         <span>
           (
           <CurrencySymbol currency={curr} />
-          {splitValueFormatted}
```

Comment:
I think this would be better named `splitValueAfterFee`

Reply:
Agreed, much clearer.